obstacledetection/src/main/java/com/caneseeproject/obstacledetection/ObstacleDetectorImpl.kt

```diff
@@ -37,7 +37,7 @@ internal class ObstacleDetectorImpl(private val odPortal: SensorPortal,
      * Send data into the cane (set the cane)
      */
     override suspend fun control(what: ODControl) {
-        send(what)
+        control(what)
```

Comment:
a duplicate function.

Reply:
something wrong is going on with me

Reply:
Finish this, and take a break. We can afford it, and you deserve it.